llvm-test-suite/cmake/opt.cmake

```diff
@@ -1,6 +1,6 @@
 # O3 = --iln --inv --red --dne --ivt --inv --dne --psh --inv --dne
 #      --red --cne --dne --pll --inv --dne --url --inv
-set(OPTFLAGS "${OPTFLAGS} -JInvariantValueRedirection -Jdne -Jivt -JInvariantValueRedirection -Jdne -JInvariantValueRedirection -Jdne -Jcne -Jdne -Jpll -JInvariantValueRedirection -Jdne -Jurl -JInvariantValueRedirection")
+set(OPTFLAGS "${OPTFLAGS} -JInvariantValueRedirection -Jred -Jdne -Jivt -JInvariantValueRedirection -Jdne -JInvariantValueRedirection -Jdne -Jred -Jcne -Jdne -Jpll -JInvariantValueRedirection -Jdne -Jurl -JInvariantValueRedirection")
```

Comment:
Could we use `-JNodeReduction` instead of `-Jred`, please? `--red` is marked deprecated in jlm-opt and will eventually disappear.

Reply:
Done